website/docs/guides/1.8.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-180-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.8.0 has a number of new and exciting features and changes.
+
+**Major Enhancements:**
+* The HashiCorp Terraform MongoDB Atlas Provider now supports [Snapshot Distribution](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule) 
+  * To learn more about this feature see here: https://www.mongodb.com/blog/post/introducing-snapshot-distribution-atlas   
+* The HashiCorp Terraform MongoDB Atlas Provider now allows users to [Programmatically Create API Keys](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/api_key) 
+  * To learn more about this feature see here: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Programmatic-API-Keys
+  *  **IMPORTANT WARNING:**  Creating, Reading, Updating, or Deleting Atlas API Keys may expose sensitive organizational secrets to Terraform State. Consider storing sensitive API Key secrets instead via the [HashiCorp Vault MongoDB Atlas Secrets Engine](https://developer.hashicorp.com/vault/docs/secrets/mongodbatlas). 
```

Comment:
Providing a bit more here will be helpful to users.  Basically first point them to Terraform best practices and then give them another option.  
```suggestion
  *  **IMPORTANT WARNING:**  Managing Atlas Programmatic API Keys (PAKs) with Terraform will expose sensitive organizational secrets in Terraform's state.  We suggest following [Terraform's best practices](https://developer.hashicorp.com/terraform/language/state/sensitive-data).  You may also want to consider managing your PAKs via a more secure method, such as the [HashiCorp Vault MongoDB Atlas Secrets Engine](https://developer.hashicorp.com/vault/docs/secrets/mongodbatlas). 
```

Reply:
updated

Reply:
also updated associated warning in terraform docs page